cmd/controller.go

```diff
@@ -54,12 +54,12 @@ func NewControllerCmd() *cobra.Command {
 				return err
 			}
 
-			kubeCfg, err := clientcmd.BuildConfigFromFlags(cfg.MasterURL, cfg.KubeConfig)
+			kubeCfg, err := kubernetes.BuildKubeClientConfig(cfg.MasterURL, cfg.KubeConfig, cfg.KubeClientTimeout)
 			if err != nil {
 				return fmt.Errorf("error building kubeConfig: %w", err)
 			}
 
-			kubeClient, err := kubernetes.NewForConfig(kubeCfg)
+			kubeClient, err := kubeClient.NewForConfig(kubeCfg)
```

Comment:
Maybe it's better to rename the alias `kubeClient` to `kubernetesClient`? This way you don't have a variable and import named `kubeClient`

Reply:
Done via 97b1771